tools/DeviceExplorer/DeviceExplorer/MainForm.cs

```diff
@@ -25,7 +27,7 @@ public partial class MainForm : Form
     {
         #region fields
         private const int MAX_TTL_VALUE = 365;
-        private const int MAX_COUNT_OF_DEVICES = 1000;
+        private const int MAX_COUNT_OF_DEVICES = 100000;
```

Comment:
>100000 [](start = 49, length = 6)

This will not take effect as the GetDevicesAsync() call caps at 1000 devices max. 
Please see note in this article:
[https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-bulk-identity-mgmt](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-bulk-identity-mgmt)

Reply:
I have more changes coming that will allow for searching for devices that are not in the first thousand, but I thank you for that article, and I have put it back for the time being 😄